Week-1/productMindset/ProductMindset.txt

```diff
@@ -0,0 +1,26 @@
+Smarto Who - Smartboard release
+SmartBoard - Intelligent and collaborative to bring together teams to create, innovate and to draw vision of prosperity.  
+On this landmark day of 02nd of April 2022, Smart Who team is proud and excited to launch a new application called SmartBoard.
+The new SmartBoard is the product to address the active feedback received from our Customers. It provides a shared platform with plethora of tools for groups of users to come together, discuss, collaborate, pen down, sketch out, and draw out the shared vision of future or to simply present crucial abstract academic concepts in a vivid, creative and palpable form.
+Along with compatibility with existing Smart Who products -SmartBoard comes with inherent links to multiple learning resources of many topics. It offers simple minimalistic and intuitive toolbars to cater to each flavour of presentation on shared media. Smartboard is not only only tailored for academic purpose, the smart minutes and maths engines makes it product for Business use as well. 
+SmartBoard sharing and content control ensures that flow of information can be disciplined if needed. SmartBoard business suite offers version control with changes added or deleted to keep track of business needs and necessary forecasting.
+�We are excited to provide the innovative solution to our customers, there win translates into our victory.� - Product Creative Manager at Smart Who
+� SmartBoard has many props, which helped keeping the attention of my students. Keeping them engaged and sharing to explain concept encouraged our students.� Dr N.Mc Fee � Head of Mathematics Department at Courage School of Mankind. 
+ 
+Check out SmartBoard today on our official website. We have many interesting offers and 30 second videos to demonstrate how Smartboard can be active part of day to day business need or to innovate or to share knowledge.
+
+FAQs
+1. Does it run on computers with limitation on specification
+
+SmartBoard base build is light, and runs smoothly on system with limited resources. Check out the requirement on our webpage. Further packages can be loaded to enable more features for business or more tailored needs.
+
+2. Do i have to use only Smart Who products with SmartBoard.
+
+SmartBoard is compatible with major conferencing and meeting planning tools. Checkout the complete list on our webpage for compatible softwares and browsers.
+
+3. How secure are my Boards when stored on Smart Who servers, does SmartBoard offers other storage means.
+
+We at Smart Who offers transparency in our services provided to our customers, security of their data is crucial for us. While, we offer storage solutions, we also offer services to help set and configure your preferred means of storage. 
+
+ 
+
```

Comment:
Please use markdown and put all content inside it.

That will give a Professional look to your content.

I've read the whole article Naveen :-) 
It is still not clear to me why an educational institute would buy "your" product...
You only gave a brief overview on what your product does...

I expected a little bit more depth... 

The goal for an educational institution would be "**to conduct classes**", "**to conduct staff/student meetings**" 
This is the **critical problem** we are trying to solve...
You mentioned collaboration... but it is not clear how your product solves the above two problems ?

I feel that the task requires us to "explain briefly" HOW our product will help educational institutions conduct classes.... track which classes have been conducted... how to plan classes (for next day/week/month)...
Take attendance of students and teachers....
Give assignments to students.
Have meetings of students and teachers.... and teachers with administration staff.... Think like this....

This is how I would think.....
Is there a central dashboard from where I (as teacher) can teach my class ?
How will I inform my students that there is an extra special class tonight from 7-8 PM ? or on weekend ? How will you notify them ?
Is there a dashboard for a headmaster(one level above teachers) to view which teacher is conducting classes "now"? 
The amount of time they took class... 
There is no clarity on "HOW" you are trying to achieve this critical task.

Need more "depth" (HOW you are trying to solve the task) on this Naveen!

Reply:
The goal for us is to give publicity that our product is the most easiest to work with in the market...

You also did not get any endorsements from an existing client ? You should have roped in an IIM or IIT for that matter (at least mentioned it...). That would bring more trust... 

This is how I thought and wrote my PR FAQ.